### DISCUSSION TITLE
Classic Navigation Block as an alternative to the Navigation Block?

### DISCUSSION THREAD

**Discussion by hagege** [WordPress/gutenberg#47381] [Ideas]
Honestly, I found creating a menu like classic themes more user-friendly. Maybe as an alternative a (new) block named “Classic Navigation Block” or something like that could be developed, which offers the options of a classic menu creation, but additionally the properties of the navigation block (color, fonts, typography, spacing, etc.) if possible. 
Although I've been working with block themes for a long time, the navigation block is actually the only block where I don't think the usability is (still) that great.
So I brought up the idea in the Slack channel #fse-outreach-experiment and @annezazu suggested I should put it up for discussion here.

**Comment by annezazu** [comment]
Can you share more @hagege about what you find easier/harder and how you imagine a classic nav block would work? Any and all details would be helpful both to improve the current nav block and to consider this idea more fully.

**Reply by hagege** [reply]
sorry for the delay in replying @annezazu: I will first do more tests with the navigation block and then document again exactly what I think is still not so ideal.

**Comment by hagege** [comment]
Hi @annezazu,
I took a detailed look at the Navigation Block this weekend and took the opportunity to write a comprehensive post (unfortunately only in German: https://haurand.com/navigation-block-wordpress-tipps-stolperstellen/). 
Many things have become clearer to me now and especially editing the navigation via settings (see here: https://haurand.com/navigation-block-wordpress-tipps-stolperstellen/#h-bearbeitung-uber-einstellungen-beim-navigation-block:~:text=ein%20Untermen%C3%BC%20einf%C3%BCgen-,Bearbeitung%20%C3%BCber%20Einstellungen%20beim%20Navigation%2DBlock,-Neu%20in%20WordPress) is indeed easier now. 
I think that although a few things should still be changed, but basically it's already great.

However, I have noticed one issue: In the properties, an editing option is shown at the following point (pencil), but editing is not possible there. I could not find a corresponding issue, but I am not sure.
![issue_navigation_1](https://user-images.githubusercontent.com/27292550/215427616-37020164-2c8d-4fa1-bcd4-0fc799d6fd06.jpg)

**Reply by annezazu** [reply]
Thanks so much for taking the time to write this up! Apologies for the delay as I was recovering. I don't speak German so am going to engage as best as I can.

>In the properties, an editing option is shown at the following point (pencil), but editing is not possible there. I could not find a corresponding issue, but I am not sure.

To focus in on this point, I'm not quite sure I follow. This could be because the experience has continued to evolve. Here's the current approach with GB 15.1:

https://user-images.githubusercontent.com/26996883/218385917-15210ac1-cb58-4f08-a896-aaaad3bca2d2.mov

**Comment by hagege** [comment]
Sorry for the delay. The issue seems to be fixed indeed.

**Comment by diegosomar** [comment]
@hagege I will post a discussion about it right now.
When I develop a website for a client, I create users with EDITOR roles, beacuse all of my clientes don't want to handle with design things. They only wants to write new posts, update page texts, images etc... Then, I create a site and the clients update only contents. They are not designers and don't want to be.

In classic themes, they can update the menu very easy, without see "design things". With FSE, to update menu, they:

1. Must be administrators;
2. Should enter in FSE area, which is very complex to non designers/webdesigners;
3. Should navigate between the blocks to update menu items. The skeleton of the FSE site can be quite daunting for some people.

In most cases, the "end-user" don't need (and don't want) to handle with design things. I love FSE and now, I develop complex sites with this tool, but the navigation always is a problem.

@hagege You suggested "Classic Navigation Block”, which is a great idea! With this block, webdesigners can specify a navigation location in the layout with Gutenberg and the end users can update the specific menu OUTSIDE the FSE editor. No risk to the end user delete or broke some thing in the layout.

Currently, this plugin already exists:
https://wordpress.org/plugins/classic-menu-block/

But is not native. I used this plugin once, but has some problems and unfortunately I don't have enough knowledge to help.
It would be wonderful if there was a native option to set classic menus in the theme.

@annezazu I think end users shouldn't edit the menu in Gutenberg. Intermediate/Advanced users only. When end users edited menus the old-fashioned way, they only edited the menu. With Gutenberg, to edit the menu, they have access to the full layout, which is very dangerous, as they can make large damages. The focus, which used to be just on the menu, is now on the entire layout. The focus should be only in the menu.

Perhaps another example to illustrate: if I want to add a post, I go to POSTS and not edit the layout to add a post. So, to edit a menu item, theoretically I would have to go to the specific area in the admin and add or remove what I wanted, without worrying about size, color, distance (margin and padding), blocks, etc.

P.S.: Sorry my english :) I'm am Brazilian.

**Comment by hagege** [comment]
@diegosomar 
Thank you for this very detailed presentation. The navigation block has definitely become better, but in fact, at least for me, there are still situations where I have my difficulties in using it, although I already do a lot with FSE. And I agree with you: The creation of the navigation in the "classic" way is certainly still easier, especially for newbies.
BTW: also my English is certainly not very good at all (I am from Germany) ;-)
